runtime/compiler/runtime/SignalHandler.c

```diff
@@ -507,6 +507,26 @@ UDATA jitPPCHandler(J9VMThread* vmThread, U_32 sigType, void* sigInfo)
 
 		exceptionTable = jitConfig->jitGetExceptionTableFromPC(vmThread, *iarPtr);
 
+		if( !exceptionTable && J9PORT_SIG_FLAG_SIGBUS == sigType ){
+		   // We might be in a jit helper routine (like arraycopy) so look at the link register as well...
+		   UDATA *lrPtr;
+		   infoType = j9sig_info(sigInfo, J9PORT_SIG_CONTROL, J9PORT_SIG_CONTROL_POWERPC_LR, &infoName, &infoValue);
+		   if (infoType != J9PORT_SIG_VALUE_ADDRESS) {
+		      return J9PORT_SIG_EXCEPTION_CONTINUE_SEARCH;
+		   }
+		   lrPtr = (UDATA *) infoValue;
+		   exceptionTable = jitConfig->jitGetExceptionTableFromPC(vmThread, *lrPtr);
+		   if (exceptionTable) {
+            vmThread->jitException = (J9Object *) (*lrPtr);  /* the lr points at the instruction after the helper call */
+#if (defined(LINUXPPC64) && !defined(__LITTLE_ENDIAN__)) || defined(AIXPPC)
+            *iarPtr = *(UDATA*) ((void *) &jitHandleInternalErrorTrap);  /* get routine address from TOC entry */
+#else
+            *iarPtr = (UDATA) ((void *) &jitHandleInternalErrorTrap);
+#endif
+            return J9PORT_SIG_EXCEPTION_CONTINUE_EXECUTION;
```

Comment:
Indentation is off for the few lines above

Reply:
I just noticed that as well. This file is using tabs rather then spaces for indenting. My editor was following that conversion until the preprocessor statements where it had no tab convention to follow from the previous line. I'll fix it to use tabs for these lines as well.